app/models/email_address.rb

```diff
@@ -23,10 +23,6 @@ def confirmed?
     confirmed_at.present?
   end
 
-  def stale_email_fingerprint?
-    Pii::Fingerprinter.stale?(email, email_fingerprint)
```

Comment:
The `stale?` method might now be unused after this method is removed?

Reply:
`Pii::Fingerprinter.stale?` is used elsewhere: https://github.com/18F/identity-idp/blob/e1b0db1c81eba880abb901c50b1885bb678ad95d/app/services/pii/cacher.rb#L61

Reply:
Huh, I missed that one. 

Okeydoke 👍